lib/target/users/user.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Target.Users.User do
+  use Ecto.Schema
+  use Pow.Ecto.Schema
+
+  schema "users" do
```

Comment:
We are missing the gender https://trello.com/c/60uY10OP/2-5-as-a-user-i-should-be-able-to-sign-up-so-my-account-is-created-in-target-mvd